spec/support/elasticsearch_helpers.rb

```diff
@@ -0,0 +1,9 @@
+# Only included in specs with elasticsearch: true
+module ElasticsearchHelpers
+  def index_documents(resources)
+    resources = [resources] unless resources.respond_to?(:each)
```

Comment:
If you want to remove the boolean you can actually do `resources  = Array.wrap(resources)` If you pass in an array it will do nothing, if you pass in a single or set of objects it will wrap them in an array/

Reply:
Great call, completely forgot about `Array.wrap`. I actually went ahead and removed that line altogether and did `Array.wrap(resources).each(:&index_to_elasticsearch_inline)` instead. Is that cool too? It's just slightly more verbose but still feels very readable to me.

Reply:
Love it!